ProcessList.c

```diff
@@ -94,7 +94,7 @@ static const char* alignedProcessFieldTitle(ProcessField field) {
 }
 
 void ProcessList_printHeader(const ProcessList* this, RichString* header) {
-   RichString_prune(header);
+   RichString_rewind(header, RichString_size(header));
```

Comment:
Why not call `RichString_setLen(header, 0)` instead?

Reply:
cause `RichString_setLen()` is currently a private function, which I'd like to keep